{
    "children": null,
    "attachments": null,
    "relations": null,
    "changesets": [
        {
            "revision": "7baf2e4d877998a30248e60955d9f13c0dcc58d9",
            "user": {
                "id": 1,
                "name": "Radegast"
            },
            "comments": "dynatimer: quick and dirty fix for lua 5.2.3, refs #609",
            "committed_on": "2014-12-05T21:29:45Z"
        },
        {
            "revision": "3dcf3eb8a138bb373c8a7e8e116fb7136ec4c1d7",
            "user": {
                "id": 140,
                "name": "Spyhawk"
            },
            "comments": "fixed attempt to index a number value error",
            "committed_on": "2016-09-03T17:04:20Z"
        }
    ],
    "journals": [
        {
            "id": 3630,
            "user": {
                "id": 13,
                "name": "RaFaL"
            },
            "notes": "Legacyconst.lua loads corectly \r\n\r\nDynamite.lua \r\nLua API: et_InitGame error running lua script: [string \"dyna.lua\"]:100: bad argument #2 to 'insert' (position out of bounds)\r\nLua API: et_RunFrame error running lua script: [string \"dyna.lua\"]:139: attempt to call field 'foreach' (a nil value)\r\n\r\nXpsave.lua closing server without an error",
            "created_on": "2014-11-08T13:45:00Z",
            "details": []
        },
        {
            "id": 3631,
            "user": {
                "id": 140,
                "name": "Spyhawk"
            },
            "notes": "See also changes done in issue #289.",
            "created_on": "2014-11-08T14:10:22Z",
            "details": []
        },
        {
            "id": 3632,
            "user": {
                "id": 1,
                "name": "Radegast"
            },
            "notes": "RaFaL wrote:\r\n> Dynamite.lua \r\n> Lua API: et_InitGame error running lua script: [string \"dyna.lua\"]:100: bad argument #2 to 'insert' (position out of bounds)\r\n> Lua API: et_RunFrame error running lua script: [string \"dyna.lua\"]:139: attempt to call field 'foreach' (a nil value)\r\ntable.insert(client_msg, i, false)  - i starts at 0 which is out of bounds since lua 5.2.2. We just have to adapt the code to count sv_maxclients at 1. Easy fix.\r\n\r\n> Xpsave.lua closing server without an error\r\nStrange. You most likely don't have sqlite module installed.\r\n",
            "created_on": "2014-11-08T15:47:37Z",
            "details": []
        },
        {
            "id": 3633,
            "user": {
                "id": 13,
                "name": "RaFaL"
            },
            "notes": "I got sqlite module installed just downloaded a new one and this \r\n\r\nLua API: G_LuaStartVM error running lua script: error loading module 'luasql.sqlite3' from file 'E:\\etlegacy\\legacy\\lualibs\\luasql\\sqlite3.dll':\r\n\tNie mo\u017cna odnale\u017a\u0107 okre\u015blonej procedury.",
            "created_on": "2014-11-08T19:18:39Z",
            "details": []
        },
        {
            "id": 3994,
            "user": {
                "id": 1,
                "name": "Radegast"
            },
            "notes": "dynatimer now works with the bundled lua",
            "created_on": "2014-12-05T21:36:40Z",
            "details": [
                {
                    "property": "attr",
                    "name": "done_ratio",
                    "old_value": "0",
                    "new_value": "10"
                }
            ]
        },
        {
            "id": 5980,
            "user": {
                "id": 140,
                "name": "Spyhawk"
            },
            "notes": "xpsave.lua gives this error at server shotdown:\r\n\r\n    Lua API: et_ShutdownGame error running lua script: [string \"xpsave.lua\"]:159: attempt to index a number value (upvalue 'cur')\r\n\r\nand\r\n\r\n    Lua API: et_ShutdownGame error running lua script: [string \"xpsave.lua\"]:160: LuaSQL: there are open cursors",
            "created_on": "2016-09-03T16:59:39Z",
            "details": [
                {
                    "property": "attr",
                    "name": "subject",
                    "old_value": "Test all Lua scripts with Lua 5.2 before next release",
                    "new_value": "Test all Lua scripts with Lua 5.3 before next release"
                }
            ]
        }
    ],
    "watchers": [
        {
            "id": 9,
            "name": "Mateos"
        }
    ],
    "time_entries": null,
    "id": 609,
    "project": {
        "id": 4,
        "name": "Lua scripts for the Legacy mod"
    },
    "tracker": {
        "id": 3,
        "name": "Task"
    },
    "status": {
        "id": 1,
        "name": "New"
    },
    "priority": {
        "id": 4,
        "name": "Normal"
    },
    "author": {
        "id": 3,
        "name": "IR4T4"
    },
    "category": {
        "id": 22,
        "name": "Lua scripts"
    },
    "fixed_version": {
        "id": 4,
        "name": "ALL"
    },
    "subject": "Test all Lua scripts with Lua 5.3 before next release",
    "description": "I've noticed some scripts in our repository don't work properly with Lua 5.2 (see dynamite.lua f.e.) when tables are used.\r\n\r\n\r\n",
    "start_date": "2014-11-05",
    "done_ratio": 10,
    "created_on": "2014-11-05T18:30:49Z",
    "updated_on": "2016-09-03T16:59:39Z"
}